projects/storefrontapp/src/app/spartacus/features/s4om/s4om-wrapper.module.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+
+import { S4omModule } from '@spartacus/s4om';
+
+@NgModule({
+  imports: [S4omModule],
+})
+export class S4OMWrapperModule {
```

Comment:
MAJOR: since there are no multiple dependencies, can we directly import S4omModule in S4omFeatureModule ?

Reply:
The file is still present - can you re-cehck